### PR TITLE
KAA-997 Add Nix guide

### PR DESCRIPTION
**Pull request by rasendubi**
/cc @forGGe

### CODE REVIEW

File: client/client-multi/client-c/docs/nix.org [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  If you want to run a single command within a shell, you can use =--run= option. For example:
+  #+begin_src sh
+    nix-shell --run make
+  #+end_src
+
+* Options
+  | Option             | Default value | Meaning                                        |
+  |--------------------+---------------+------------------------------------------------|
+  | posixSupport       | true          | Host build with gcc. Goes to =build-posix=.    |
+  | clangSupport       | true          | Host build with clang. Goest to =build-clang=. |
```
Comment (rasendubi) [inline client/client-multi/client-c/docs/nix.org]: Typo: `Goest`

File: client/client-multi/client-c/docs/nix.org [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  | withWerror         | false         | Enable =-Werror= for all builds.               |
+
+  You can override any option with the following command:
+  #+begin_src sh
+    nix-shell --arg optionName value
+  #+end_src
+
+  For example:
+  #+begin_src sh
+    nix-shell --arg withWerror false
```
Comment (rasendubi) [inline client/client-multi/client-c/docs/nix.org]: It's better to use `true` here, as it's not default.

File: client/client-multi/client-c/docs/nix.org [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  If you don't trust piping shell scripts from the Internet (and you shouldn't), feel free to examine the script or use alternate setup. (Though, you should google it up yourself.)
+
+* Install all dependencies and enter the shell
+  The first time you enter shell environment, the Nix will install all dependencies needed for development.
+  #+begin_src sh
+    nix-shell
+  #+end_src
+
+  As CC3200 SDK is not freely available, nix-shell will abort and ask you to download the file manually and add it to nix-store -- follow instructions. Then re-run the command above.
```
Comment (forGGe) [inline client/client-multi/client-c/docs/nix.org:24]: Probably it is worth to add here example of error and explicitly give a command that will add cc32xx SDK to the nix.

**Comment by forGGe** [comment]
Reviewed.

File: client/client-multi/client-c/docs/nix.org (hunk 3 above)
Comment (rasendubi) [inline client/client-multi/client-c/docs/nix.org:24]: I don't think that's needed. Nix prints both the reason and how to fix it.

Comment (forGGe) [inline client/client-multi/client-c/docs/nix.org:24]: Per discussion: agreed.

**Comment by forGGe** [comment]
+

**Pull request merged**